### PR TITLE
Book

### PR DESCRIPTION
**Pull request by franciscoabalan**
## Summary
Create book model

## Trellocard
https://trello.com/c/5Vr1k6y9/18-crear-modelo-de-book

## Screenshot
Captura de Pantalla 2019-10-03 a la(s) 12 52 25

### CODE REVIEW

File: app/models/book.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```
Comment (joredudiaz) [inline app/models/book.rb:1]: Please validate the presence of the fields.

File: test/fixtures/books.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (aleochoam) [inline test/fixtures/books.yml]: You can delete this file, we won't need it

File: test/models/book_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (aleochoam) [inline test/models/book_test.rb]: Same here

File: app/models/book.rb (hunk 1 above)
Comment (franciscoabalan) [reply app/models/book.rb:1]: Done

File: test/fixtures/books.yml (hunk 2 above)
Comment (franciscoabalan) [reply test/fixtures/books.yml]: Done

File: test/models/book_test.rb (hunk 3 above)
Comment (franciscoabalan) [reply test/models/book_test.rb]: Done

**Review by aleochoam — APPROVED** [review]

**Pull request merged**